awx/main/expect/isolated_manager.py

```diff
@@ -319,58 +248,51 @@ def health_check(cls, instance_qs, awx_application_version):
          - clean up orphaned private files
         Performs save on each instance to update its capacity.
         '''
-        hostname_string = ''
-        for instance in instance_qs:
-            hostname_string += '{},'.format(instance.hostname)
-        args = cls._build_args('heartbeat_isolated.yml', hostname_string)
-        args.extend(['--forks', str(len(instance_qs))])
-        env = cls._base_management_env()
+        # TODO: runner doesn't have a --forks arg
+        #args.extend(['--forks', str(len(instance_qs))])
```

Comment:
ansible-runner doesn't support this in its programmatic interface, so I've got to figure out a way to pass it in.

Reply:
Talked to @chrismeyersfsu; I'll do this later when it's supported a new ansible-runner:

https://github.com/ansible/ansible-runner/pull/241

Reply:
This is now merged